server/src/services/library.service.ts

```diff
@@ -453,6 +453,14 @@ export class LibraryService extends EventEmitter {
         `File modification time has changed, re-importing asset: ${assetPath}. Old mtime: ${existingAssetEntity.fileModifiedAt}. New mtime: ${stats.mtime}`,
       );
       doRefresh = true;
+    } else if (existingAssetEntity.originalFileName !== parse(assetPath).base) {
```

Comment:
Nit: the asset path is parsed twice. You can move it to a variable to reuse it.

Reply:
Done, thanks